esp/esp/program/modules/handlers/teacheracknowledgementmodule.py

```diff
@@ -26,7 +26,8 @@ def module_properties(cls):
             "admin_title": "Teacher Acknowledgement",
             "link_title": "Teacher Acknowledgement",
             "module_type": "teach",
-            "required": True,
+            "required": False,
+            'choosable': 1,
```

Comment:
It's a judgment call, I guess. I think it's good for teachers to check the box, and I was trying to keep the list of questions admins have to answer to a minimum, but I don't feel super strongly. @kkbrum want to weigh in?

Reply:
Having it default to included makes sense to me since I think teachers should always have to agree to show up and if a program really doesnt want it they can always remove the module later

Reply:
I went back to see the content of the default "acknowledgment", and I guess it's pretty generic enough to include for most programs. The student acknowledgment module seems a little more specific/less generic. Also, why did you change `required` to `False`?

Reply:
so that teachers could remove it if they wanted to

Reply:
I don't follow. `required` defines whether or not a module needs to be completed before moving on to the next module.

Reply:
fixed by commit ddccee4

Reply:
If you're including this module by default, it should be required. There's no point in having some sort of agreement and not having it be required.

Reply:
changed in cdeabbf